core/nativeaot/HelloWorld/README.md

```diff
@@ -17,27 +17,7 @@ This will create a simple Hello World console app in `Program.cs` and associated
 
 ## Add NativeAOT to your project
 
-To use NativeAOT with your project, you need to add a reference to the ILCompiler NuGet package that contains the NativeAOT ahead of time compiler and runtime.
-For the compiler to work, it first needs to be added to your project.
-
-In your shell/command prompt navigate to the root directory of your project and run the command:
-
-```bash
-> dotnet new nugetconfig
-```
-
-This will add a nuget.config file to your application. Open the file and in the ``<packageSources>`` element under ``<clear/>`` add the following:
-
-```xml
-<add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
-<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-```
-
-Once you've added the package source, add a reference to the compiler by running the following command:
-
-```bash
-> dotnet add package Microsoft.DotNet.ILCompiler -v 7.0.0-*
-```
+Add `<PublishAot>true</PublishAot>` property to your project file. This will produce a native AOT app and shows PublishAot compatibility warnings during build.
```

Comment:
```suggestion
Add `<PublishAot>true</PublishAot>` property to your project file. This will produce a native AOT app and show any potential compatibility warnings during the publish process.
```